src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -1360,6 +1360,40 @@ def helm_install_release(
                 "'arcConfigEndpoint' doesn't exist under 'dataplaneEndpoints' in the ARM metadata."
             )
 
+    # Add overrides for AGC Scenario
+    if cloud_name.lower() == "ussec" or cloud_name.lower() == "usnat":
+        logger.debug("Adding AGC scenario overrides.")
+
+        arm_metadata_endpoint_array = arm_metadata["authentication"]["loginEndpoint"].strip("/").split(".")
+        
+        if len(arm_metadata_endpoint_array) < 4:
+        raise CLIInternalError("Unexpected loginEndpoint format for AGC")
+
+        cloud_suffix = arm_metadata_endpoint_array[3]
+        endpoint_suffix = arm_metadata_endpoint_array[2] + "." + arm_metadata_endpoint_array[3]
+        if (cloud_name.lower() == "usnat"):
+            cloud_suffix = arm_metadata_endpoint_array[2] + "." + arm_metadata_endpoint_array[3] + "." + arm_metadata_endpoint_array[4]
+            endpoint_suffix = cloud_suffix
+
+        cmd_helm_install.extend(
+            [
+                "--set",
```

Comment:
can we detect these config values at a single location?

Reply:
listing the urls would not be feasible but the logic can be moved to a common place.  please correct me if not so